### PR TITLE
Apply linters, use CSS selector and refactored index.html

### PR DESCRIPTION
**Pull request by adamilare**
Following the given requirements, I;
- Use semantic HTML tags;
- Use CSS selectors;
- Use CSS box model;
- Use best HTML best practices in code.
- Apply linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,18 +2,15 @@
...
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
-
   </head>
 
   <body>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-<!-- 
-    <div>
-      <h3>Hello World, we are software developers!</h3>
-    </div>  -->
```
Comment (adamilare) [inline index.html:16]: It is advisable not to leave comments in code. So please remove comments in your code,

File: index.html [hunk 2]
```diff
@@ -24,55 +21,52 @@ <h1 id="title" class="title">
...
-        <label>
           <input name="check-opt" value="check3" type="checkbox">Check 3</input>
-        </label>
+        </div>
       </div>
       <div class="form-group">
-        <p><label for="comments">Additional Comments:</label></p>
-        <textarea id="w3review" name="w3review" rows="4" cols="50"></textarea>
+        <label for="comments">Additional Comments:</label>
+        <textarea class="input" id="w3review" name="w3review" rows="4" cols="50"></textarea>
```
Comment (adamilare) [inline index.html]: It is best practice to name your classes or id differently from the HTML tags or semantics

**Review by adamilare — COMMENTED** [review]
Hi @adamilare, 

Your project is complete!  There is nothing else to say other than... it's time to merge it ♻️ : 
Congratulations! 🎉 

## Highlights

- Followed Github flow ✔️
- Professional README file. ✔️
- Commit messages should be meaningful. ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (ABDULALI3468) [inline index.html]: > * [ ]  Good Job for using the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) But I recommend you to consider following example for this specific case. You can rewrite above as follow:
 For Example: -
> * "Please consider naming your classes and ids differently from the HTML tags as it is considered to be a best practice for semantics.

> In your above comment you only described the `reason` but always consider telling them what should they do, also.

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @adamilare , 



Literally did exceptional work 👏 but there are some small changes that you need to review. Please consider checking them.


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

_**Check the comment below for the Required Changes.**_

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @adamilare, 👋 

Your project is complete! Great job for following the project requirement🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**